applications/system/js_app/modules/js_i2c.c

```diff
@@ -0,0 +1,254 @@
+#include "../js_modules.h"
+#include <furi_hal_i2c.h>
+
+typedef struct {
+    uint32_t addr;
+} JsI2cInst;
```

Comment:
all the functions seem to use the `addr` provided without state implications, is this state struct required?